src/framework/ui/view/abstractmenumodel.cpp

```diff
@@ -73,59 +75,51 @@ MenuItem& AbstractMenuModel::findMenu(const ActionCode& subitemsActionCode)
     return menu(m_items, subitemsActionCode);
 }
 
-MenuItem AbstractMenuModel::makeMenu(const std::string& title, const MenuItemList& actions, bool enabled,
+MenuItem AbstractMenuModel::makeMenu(const QString& title, const MenuItemList& items, bool enabled,
                                      const ActionCode& menuActionCode) const
 {
     MenuItem item;
     item.code = menuActionCode;
     item.title = title;
-    item.subitems = actions;
-    item.enabled = enabled;
+    item.subitems = items;
+    item.state.enabled = enabled;
     return item;
 }
 
 MenuItem AbstractMenuModel::makeAction(const ActionCode& actionCode) const
 {
-    ActionItem action = actionsRegister()->action(actionCode);
+    const UiAction& action = uiactionsRegister()->action(actionCode);
+//    IF_ASSERT_FAILED(action.isValid()) {
```

Comment:
Please, remove (or fix this code)

Reply:
Hmm, this is a dilemma, I believe that there should be an assertion, but in fact, now there are items in the menu for which there are no UI actions